### PR TITLE
Documentation: Document a reliable way how to set the display timeout.

### PR DESCRIPTION
**Pull request by matsimon**
This was the easiest way to modify the defaults off, in this case, Fedora 25.

### CODE REVIEW

**Comment by matsimon** [comment]
Forgot to assign this to anyone ;-)

File: doc/virtesk-tc-tools.rst [hunk 1]
```diff
@@ -434,3 +434,16 @@ This was seen consistently on a combination with systems running on Intel H55 ch
 using the integrated HD Graphics on Fedora 25 with Linux kernel 4.8.5.
 
 In this case a known workaround was to append "--real-mode" to the kexec loading parameter (kexec -l).
+
+Inadequate screensaver timeout
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Situation: Depending on the used versions of Fluxbox, LXDM and X.org, the
+default timeout for screen powerdown may not equal your requirements. - Also
+default values do vary in between disttribution versions thus you might want
```
Comment (karras) [inline doc/virtesk-tc-tools.rst:443]: s/in between/inbetween/
s/disttribution/distribution

File: doc/virtesk-tc-tools.rst [hunk 2]
```diff
@@ -434,3 +434,16 @@ This was seen consistently on a combination with systems running on Intel H55 ch
 using the integrated HD Graphics on Fedora 25 with Linux kernel 4.8.5.
 
 In this case a known workaround was to append "--real-mode" to the kexec loading parameter (kexec -l).
+
+Inadequate screensaver timeout
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Situation: Depending on the used versions of Fluxbox, LXDM and X.org, the
+default timeout for screen powerdown may not equal your requirements. - Also
```
Comment (karras) [inline doc/virtesk-tc-tools.rst:442]: Remove the "-"...

File: doc/virtesk-tc-tools.rst [hunk 3]
```diff
@@ -434,3 +434,16 @@ This was seen consistently on a combination with systems running on Intel H55 ch
...
+default values do vary in between disttribution versions thus you might want
+to adapt it to your needs.
+
+A reliable to customize that timeout is to add the following line in the
```
Comment (karras) [inline doc/virtesk-tc-tools.rst:446]: s/reliable/reliable way/

**Review by karras — CHANGES_REQUESTED** [review]

**Comment by karras** [comment]
Project archived

**Pull request closed without merging**